sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Idk all the answers, just tried to do something fun. We can delete this branch and just deal with it next week.